test/fixtures/simple_example/versions.tf

```diff
@@ -17,3 +17,11 @@
 terraform {
   required_version = ">= 0.12"
 }
+
+provider "google" {
```

Comment:
Put these on the example, not the fixture.

Reply:
done